create_derivatives/routines.py

```diff
@@ -1,12 +1,13 @@
+from os.path import join
```

Comment:
Added this comment on @bonniegee's PRs as well but let's use `pathlib` exclusively in this app. See the docs: https://docs.python.org/3.6/library/pathlib.html